debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfigTest.java

```diff
@@ -149,6 +155,32 @@ public void testPrimaryKeyRecordValueDoesNotRequirePrimaryKeyFields() {
         assertThat(config.validateAndRecord(List.of(JdbcSinkConnectorConfig.INSERT_MODE_FIELD), LOGGER::error)).isTrue();
     }
 
+    @Test
+    public void testDeprecatedTableNamingStrategy() {
```

Comment:
@mfvitale @jpechane @Naros I have added a special test case here to check the deprecate behavior is working as expected, returns the proper deprecation messages and the fallback behavior is working in all directions.
let me know what you think!

Reply:
Now we can say that it works without relying on hope :wink:

Reply:
I would not go that far because the previous tests were already using the old config options and were still working like expected. After re-adding the old API which is now calling the new implementation it made sense to add this definitely more conscious test. Agreed so far! ;)